scripts/azureml-assets/azureml/assets/environment/build.py

```diff
@@ -72,9 +72,10 @@ def create_acr_task(image_name: str,
                         f"$Registry/{image_name} "
                         f"curl -sSfL -o trivy.deb {trivy_url} && "
                         "dpkg -i trivy.deb && "
-                        "trivy fs --scanners vuln /"
+                        f"aquasec/trivy image --scanners vuln --ignore-unfixed $Registry/{image_name}"

```

Comment:
Please try replacing this entire section with:
`'cmd': f"aquasec/trivy image --scanners vuln --ignore-unfixed $Registry/{image_name}",`